include/rtc.h

```diff
@@ -17,21 +17,75 @@
 
 #define RTC_ERR_FLAG_MASK      0x0FF0
 
-#define MORNING_EVO_HOUR_BEGIN 6
-#define MORNING_EVO_HOUR_END   10
+//Morning/evening don't exist in Gen 3, so they use Gen 4/5's values
```

Comment:
This is currently a placeholder so the code doesn't break. If anyone has any better ideas on how to implement evening in Gen 3/4 and morning in Gen 3, please let me know

Reply:
Morning time doesn't affect anything by default in Gen 3, and the `IsBetweenHours()` always returns false if the starting and ending time are the same, so you could just dummy them out by setting them to zero so that it quite literally just doesn't exist in Gen 3. I think you could dummy out evening in Gen 3 too, and while that would indeed prevent (Own Tempo) Rockruff from evolving, it's not set to Gen 3 by default anyway (and FRLG also didn't support any time-based evolutions anyway), so I don't consider it to be an issue personally.